dao/src/main/java/se/fortnox/reactivewizard/db/DbProxy.java

```diff
@@ -54,6 +55,20 @@ protected DbProxy(ReactiveStatementFactory reactiveStatementFactory,
         this.handlers = handlers;
     }
 
+    /**
+     * @deprecated Use {@link #DbProxy(DatabaseConfig, ConnectionProvider)} instead
```

Comment:
I wonder why the replacement link is referring the two arguments constructor instead of the one that takes all three arguments omitting only `JsonSerializerFactory`? I mean this constructor: 
```
DbProxy(DatabaseConfig databaseConfig, ConnectionProvider connectionProvider,
        DbStatementFactoryFactory dbStatementFactoryFactory)
```